### ISSUE TITLE
3.7.0 causes Sass::SyntaxError parsing bootstrap-sass styles

### ISSUE THREAD

**Issue by bjeanes** [sass/ruby-sass#94]
I'm getting the following exception when loading certain assets after upgrading to the latest release of ruby-sass:

```
ActionView::Template::Error:
  | Invalid CSS after "...top: 4px solid ": expected ";", was "\	;"
```

I have tried grepping my entire codebase and all gems that I depend on for this CSS but cannot find a likely suspect:

```
$ bundle show --paths | xargs ag 'top: 4px solid' .
/Users/bjeanes/.gem/ruby/2.5.3/gems/crass-1.0.4/test/support/serialization/bootstrap.css
3209:  border-top: 4px solid #000000;
```

The only match, as shown above, seems to be from a _test_ file in the crass gem. This might be what's happening, as I only get this syntax error in test mode (or when pre-compiling with `RAILS_ENV=test`), however I don't see how that CSS is invalid, why it would be parsed by Sass regardless (as it's `.css`), nor how the breakage would relate to anything mentioned in the change log for this release.

Any ideas?

(N.B.: I'm aware that I should be moving so `sassc`, but currently blocked by https://github.com/FortAwesome/font-awesome-sass/issues/166 or changing our entire site to use FontAwesome 5 icons—a non-trivial amount of work).

**Comment by bjeanes** [comment]
It seems to be failing on [this line in `bootstrap-sass`](https://github.com/twbs/bootstrap-sass/blob/5d6b2ebba0c2a5885ce2f0e01e9218db3d3b5e47/assets/stylesheets/bootstrap/_dropdowns.scss#L14):

``` sass
border-top:   $caret-width-base solid \9; // IE8
```

I have no explanation for why this only happens in test mode, though.

**Comment by bjeanes** [comment]
> You might want to look into your compiling stack to figure out why that's happening.

Hmm... Our compiling stack is a pretty stock-standard Rails app in this respect I _think_, so I'm not sure what that could be...

I also cannot account for the fact that this is only happening in test environment, but that does seem to be an indicator that the way in which assets are compiled varies between my environments. I just haven't been able to find any configuration on our part that would have this effect...

**Comment by rene-demonsters** [comment]
I ended up running in the same issue, but only when running rake assets:precompile in production. Downgrading sass fixed it for me: https://stackoverflow.com/questions/53183486/sasssyntaxerror-preventing-heroku-rails-app-from-precompiling/53191970#53191970

**Comment by dylanfisher** [comment]
Hi @nex3 thanks for the quick fix for the bug showing up in Bootstrap. I am using another gem that causes a similar issue related to `\` character:

https://github.com/tors/jquery-fileupload-rails/blob/ec604cdaf5daa7bc019ba0d605f9d70e06f7494c/app/assets/stylesheets/jquery.fileupload.scss#L31

`ActionView::Template::Error (Invalid CSS after "screen": expected media query list, was "\"):`

I can just override this file since I don't need IE < 8 support, but this line didn't cause a problem before Sass 3.7.0.

**Comment by nex3** [comment]
@dylanfisher Can you open a new issue for that?
